src/presentation/http/router/note.test.ts

```diff
@@ -576,7 +548,19 @@ describe('Note API', () => {
   });
 
   describe('POST /note', () => {
-    test('Should correctly save relation to parent note if parentId passed', async () => {
+    /* Should correctly save note tools */
+    test.each([
+      {
+        noteTools: [
+          { 'header' : 1 },
+          { 'list': 3 },
```

Comment:
lets move number to constants to make it clear where they came from

or event better 

```ts
const headerTool = {
  id: 1,
  name: 'header'
}

const listTool = {
  id: 3,
  name: 'list'
}

tests.each([
 { 
   noteTools: [
     {[headerTool.name]: headerTool.id },
     {[listTool.name]: listTool.id }
   ]
])
```